doc/Known_issues/Service_restart_workarounds.md

```diff
@@ -0,0 +1,42 @@
+Windows Server 2016 with Docker 18.09 has several issues with associating HNS networks and Docker networks.
+
+- Restarting Docker service causes Docker networks to lose   their association to corresponding HNS networks, if they   were created using custom network plugins.
+ It is caused by lack of information regarding network plugin in HNS itself.
+ Networks created using Contrail CNM plugin are registered in HNS with type `transparent`.
+ During a restart, Docker removes networks created with custom plugins.
+
+## Problems
+
+### Docker restart
+
+- `docker network prune`: using this command after docker restart isn't supported.
+ Because Docker fills it's network database in incorrect way,
+ using this command could delete a HNS root network which could lead to e.g. BSOD.
+
+### Compute node reboot
+
+- container deletion: Docker doesn't allow deleting a network which has container attached to it, regardless of the container's state.
+ As the networks states are invalid after reboot, both networks and containers need to be deleted.
+
+## Workaround: contrail-autostart
+
+### Description
+
+contrail-autostart is a script invoked at the startup of Windows whose job is to workaround weird Docker for Windows behaviour related to rebooting.
+
+contrail-autostart does the following:
```

Comment:
This list is not formatted properly in mkdocs. There should be an empty line before an enumeration and nested lists should be indented with 4 spaces (or a multiply of 4 for deeply nested).

You should probably define it like this:

```
contrail-autostart does the following:

- Starts docker and contrail services:
    - Order:
        - Docker.
```